run-tests.sh

```diff
@@ -0,0 +1,19 @@
+JACKD_CMD_ARGS="-r -d dummy -r 48000 -p 8192"
+
+killall -SIGKILL jackd mod-host || true
```

Comment:
this kill is not needed, and dont think it should be used.
it is on the CI file only because of the use of "gdb" for debugging output, we need to kill previous instance that gets stuck.